manifests/n/namazso/OpenHashTab/v3.0.2/namazso.OpenHashTab.locale.en-US.yaml

```diff
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.5-1-25136-1000
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: namazso.OpenHashTab
+PackageVersion: v3.0.2
+PackageLocale: en-US
+Publisher: namazso
+PublisherUrl: https://namazso.eu/
+PublisherSupportUrl: https://github.com/namazso/OpenHashTab/issues
+# PrivacyUrl: 
+Author: namazso
+PackageName: OpenHashTab
+PackageUrl: https://github.com/namazso/OpenHashTab
+License: GPL-3.0 License
+LicenseUrl: https://github.com/namazso/OpenHashTab/blob/master/license.installer.txt
+Copyright: Copyright (c) 2019-2021 namazso
+CopyrightUrl: https://github.com/namazso/OpenHashTab#license
```

Comment:
This could also use the specific revision: https://github.com/namazso/OpenHashTab/tree/v3.0.2#license